warp/Network/Wai/Handler/Warp/Settings.hs

```diff
@@ -235,4 +236,4 @@ exceptionResponseForDebug :: SomeException -> Response
 exceptionResponseForDebug e =
     responseBuilder H.internalServerError500
                     [(H.hContentType, "text/plain; charset=utf-8")]
-                    $ byteString . C8.pack $ "Exception: " ++ show e
+                    $ byteString . TE.encodeUtf8 . T.pack $ "Exception: " ++ show e
```

Comment:
Good catch, nice improvement! How about using [`encodeUtf8Builder`](https://www.stackage.org/haddock/lts-16.28/text-1.2.4.0/Data-Text-Encoding.html#v:encodeUtf8Builder) instead?

Reply:
Sure, changed to `encodeUtf8Builder`.